source/Conversation.cpp

```diff
@@ -127,12 +127,12 @@ void Conversation::Load(const DataNode &node, const string &missionName)
 
 				// Store the text of this choice. By default, the choice will
 				// just bring you to the next node in the script.
-				nodes.back().data.emplace_back(grand.Token(0), nodes.size());
-				nodes.back().data.back().first += '\n';
+				nodes.back().elements.emplace_back(grand.Token(0), nodes.size());
+				nodes.back().elements.back().text += '\n';
```

Comment:
I wrote it that way because of old habits. I honestly misread the code slightly until just now, in a way that made me think there was a (very small) performance penalty to doing it the other way.